### ISSUE TITLE
Fastboot : kernel panic

### ISSUE THREAD

**Issue by trezel** [Thinstation/thinstation#430]
Hi,
I have a working thinstation build, clients booting on pxe.
I try now to speed boot with fastboot, but it crashes with kernel panic message.

 - In build.conf, I have :
param fastboot true
package autonet
param initrdcmd "gzip -9"   ## instead of squashfs

 - In thinstation.conf.buildtime, I added :
FASTBOOT_URL="http://serverIPaddr/thinstation/"

 - PXE parameters are :
KERNEL /test/vmlinuz-ts6-fastboot
APPEND initrd=/test/initrd-ts6-fastboot console=tty1 LM=3 FASTBOOT_URL="http://serverIPaddr/thinstation"

(of course serverIPaddr should be replaced with my web server IPaddress)
wget http://serverIPaddr/thinstation/lib.squash works fine from another client on the same network.
Any idea?... (I precise again that everything works fine if I boot this thinstation build without fastboot options described above)

**Comment by sqnoc** [comment]
Hi,
Are you disabled package overlayfs ?

**Comment by trezel** [comment]
Hi,

No, I didn't. I hadn't had to do it in previous version...
Since I posted this issue, I've decided to use another way to "fastboot" : I migrated my pxe server to ipxe. This way, vmlinuz and initrd are loaded using http instead of tftp, and it works just fine.
(this is far not the only enhancement ipxe allow : ipxe.org for those who don't know it /are curious).
If I got some time left soon, I'll test fastboot without overlayfs to complete this post and confirm if your solution works or not.

Thanks
ET
